.github/workflows/ci-android-emulator-tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Checkout the repository
         uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
       - name: Download and Setup the Android NDK
-        uses: nttld/setup-ndk@8c3b609ff4d54576ea420551943fd34b4d03b0dc # v1.2.0
+        uses: nttld/setup-ndk@dbacc5871a0fac6eef9a09d2ca86bc8bf79432c3 # v1.3.1
```

Comment:
I verified this is the hash for the v1.3.1 release:
https://github.com/nttld/setup-ndk/commit/dbacc5871a0fac6eef9a09d2ca86bc8bf79432c3